agent/api/task/task.go

```diff
@@ -218,7 +217,8 @@ type Task struct {
 	// credentialsID is used to set the CredentialsId field for the
 	// IAMRoleCredentials object associated with the task. This id can be
 	// used to look up the credentials for task in the credentials manager
-	credentialsID string
+	credentialsID                string
+	credentialsRelativeURIUnsafe string
```

Comment:
lets make sure this doesn't get serialized to disk in the config

Reply:
yeah, checked and it's not in state file